deps/fpconv/README.md

```diff
@@ -0,0 +1,9 @@
+libfpconv
+
+----------------------------------------------
+
+Fast and accurate double to string conversion based on Florian Loitsch's Grisu-algorithm[1].
+
+This port contains a subset of the 'C' version of Fast and accurate double to string conversion based on Florian Loitsch's Grisu-algorithm available at [github.com/night-shift/fpconv](https://github.com/night-shift/fpconv)).
```

Comment:
Is it a subset of the files from that repo?

It would be useful to include some info about how to uodate the lib in the future. If some change is needed, if could be included as a patch file.

Reply:
agree. will prepare the initial patch @zuiderkwast